globus_sdk/transfer/paging.py

```diff
@@ -180,6 +183,19 @@ def __init__(
             # way of making sure that it's clear
             self.generator = None
 
+    @property
+    def limit_less_than_available_results(self):
+        """
+        Indicates that the Transfer API had more results
+        available than were requested by way of the `limit` parameter.
+
+        Note: this will always be false until the iterator containing the
+        results has been exhausted.
```

Comment:
The phrasing you've used is accurate, but I can't shake the feeling that this is confusing.

The issue is that the iterator can be exhausted _without_ setting this to True if `num_results=None`, but if the API is limiting our results, you still got "less than available".

But I don't see a way to make it better than what you've done.

Reply:
Yeah, I'm not invested in this name in any way. I went through a couple and landed on this for lack of anything better.

Reply:
Not a fan of `limit_param_was_obeyed_and_thats_why_you_got_fewer_results_than_available_but_if_false_could_indicate_that_other_limiting_factor_was_at_play`?
That's my first instinct, obviously. Explicit is better than implicit.